scripts/snakemake_sophie_format_template_experiment_hogan.R

```diff
@@ -0,0 +1,122 @@
+library(readr)
+library(dplyr)
+library(tibble)
+
+# read in counts and metadata ---------------------------------------------
+
+# set strain using wildcard so that proper set of control samples can be extracted
+strain <- unlist(snakemake@wildcards[['strain']])
+
+# read in hogan lab sample counts
+#counts <- read_csv("outputs/combined_new/num_reads_pao1.csv")
```

Comment:
I usually leave these in so that when I go to debug or change the functionality of a script, I don't have to think about which files to test it in and re-type their names.